lib/puppet/provider/ios_acl_entry/ios.rb

```diff
@@ -0,0 +1,453 @@
+          instance[:source_eq] << split_output.shift
+        end
+        next_token = split_output.shift
+      elsif next_token == 'gt'
+        instance[:source_gt] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'lt'
+        instance[:source_lt] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'neq'
+        instance[:source_neq] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'portgroup'
+        instance[:source_portgroup] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'range'
+        instance[:source_range] = []
+        instance[:source_range] << split_output.shift
+        instance[:source_range] << split_output.shift
+        next_token = split_output.shift
+      end
+      if next_token == 'any'
+        instance[:destination_address_any] = true
+      elsif next_token == 'host'
+        instance[:destination_address_host] = split_output.shift
+      elsif next_token == 'addrgroup'
+        instance[:destination_address_group] = split_output.shift
+      elsif split_output[0] =~ %r{(^\d+[.]\d+[.]\d+[.]\d+$)}
+        instance[:destination_address] = next_token
+        if split_output[0] =~ %r{(^\d+[.]\d+[.]\d+[.]\d+$)}
+          instance[:destination_address_wildcard_mask] = split_output.shift
+        end
+      end
+
+      next_token = split_output.shift
+      if next_token == 'eq'
+        instance[:destination_eq] = []
+        while split_output[0] && (split_output[0] =~ %r{(\d+)} || port_types.include?(split_output[0]))
+          instance[:destination_eq] << split_output.shift
+        end
+        next_token = split_output.shift
+      elsif next_token == 'gt'
+        instance[:destination_gt] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'lt'
+        instance[:destination_lt] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'neq'
+        instance[:destination_neq] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'portgroup'
+        instance[:destination_portgroup] = split_output.shift
+        next_token = split_output.shift
+      elsif next_token == 'range'
+        instance[:destination_range] = []
+        instance[:destination_range] << split_output.shift
+        instance[:destination_range] << split_output.shift
+        next_token = split_output.shift
+      end
+    end
+
+    until next_token.nil?
+      if next_token == 'ack'
+        instance[:ack] = true
+      elsif next_token == 'dscp'
+        instance[:dscp] = split_output.shift
+
+      elsif next_token == 'fin'
+        instance[:fin] = true
+      elsif next_token == 'fragments'
+        instance[:fragments] = true
+
+      elsif next_token == 'log'
+        instance[:log] = true
+      elsif next_token == 'log_input'
```

Comment:
This token should be `log-input`

Reply:
Good catch. Amended a unit test to include this.